worldedit-core/src/main/java/com/fastasyncworldedit/core/util/image/ImageUtil.java

```diff
@@ -229,7 +230,9 @@ public static URI getImageURI(String arg) throws InputParseException {
                 if (arg.contains("imgur.com") && !arg.contains("i.imgur.com")) {
                     arg = "https://i.imgur.com/" + arg.split("imgur.com/")[1] + ".png";
                 }
-                return new URL(arg).toURI();
+                URL url = new URL(arg);
```

Comment:
This constructor is deprecated in Java 20, maybe it makes sense to direclty construct the URI and also let the checkImageHost take a URI?

Reply:
OIC, yeah probably worth that then